packages/ui/cypress/utils/waitForMultisigLength.ts

```diff
@@ -0,0 +1,13 @@
+import { multisigPage } from '../support/page-objects/multisigPage'
+import { topMenuItems } from '../support/page-objects/topMenuItems'
+
+export const waitForMultisigLength = (length: number) =>
+  cy.waitUntil(() => {
+    multisigPage.accountHeader().click()
```

Comment:
this is doing a click outside basically, so that if any dropdown is open, it's close it.

Reply:
Sounds good, might be worth adding a small comment there about that so we remember why we are doing that.